test/integration_test/migration_test.go

```diff
@@ -568,3 +575,87 @@ func migrationScheduleTest(
 	_, err = task.DoRetryWithTimeout(f, defaultWaitTimeout, defaultWaitInterval)
 	require.NoError(t, err, "migrations for schedules have not been deleted")
 }
+
+func migrationScaleTest(t *testing.T) {
+	ctxs := make([]*scheduler.Context, migrationScaleCount)
+	var allMigrations []*v1alpha1.Migration
+	var err error
+	// Reset config in case of error
+	defer func() {
+		err = setRemoteConfig("")
+		require.NoError(t, err, "Error resetting remote config")
+	}()
+	migrationKey := "mysql-migration"
+	migrationAppKey := "mysql-1-pvc"
+	includeResourcesFlag := true
+
+	for i := 1; i <= migrationScaleCount; i++ {
+		currCtxs, err := schedulerDriver.Schedule(migrationKey+"-"+strconv.Itoa(i),
```

Comment:
Can you use `triggerMigration()` here? You should be able to do the same thing with that.

Reply:
As discussed, trigger migrations will cause same migration to be created everytime, so need a separate function to create a new migration everytime.